packages/functional-tests/tests/signin/signinCached.spec.ts

```diff
@@ -28,7 +25,6 @@ test.describe('severity-2 #smoke', () => {
 
     test.afterEach(async ({ target }) => {
       test.slow(); //The cleanup was timing out and exceeding 3000ms
-      await syncBrowserPages.browser?.close();
       if (email) {
         // Cleanup any accounts created during the test
         await target.auth.accountDestroy(email, password);
```

Comment:
Unrelated to the work in this PR - but noticing that we don't seem to have cleanup for the account created with `email2`. 👀

Reply:
😬 You are right. I can patch in my PR, but hopefully [FXA-8984](https://mozilla-hub.atlassian.net/browse/FXA-8984) will lead to a better solution